sympy/printing/pretty/pretty_symbology.py

```diff
@@ -663,3 +663,28 @@ def is_subscriptable_in_unicode(subscript):
 
     """
     return all(character in sub for character in subscript)
+
+
+def center_pad(wstring, wtarget):
+    """
+    Returns the padding strings necessary for centering a string of line_width wstring
+    to a string of line_width wtarget.
+
+    The line_width wstring should always be less or equal to wtarget.
+    """
+    wdelta = wtarget - wstring
+    assert wdelta >= 0
+
+    wleft = wdelta // 2
```

Comment:
I think to preserve the existing master behavior we could do `wleft = sum(divmod(wdelta, 2))`, i.e. `ceiling(wdelta/2`. Does anyone have a preference?